Frontend/ui-customer/src/Screens/NammaSafetyFlow/Components/HelperViews.purs

```diff
@@ -0,0 +1,40 @@
+module Screens.NammaSafetyFlow.Components.HelperViews where
+
+import Prelude
+import PrestoDOM
+
+import Common.Types.App (LazyCheck(..))
+import Effect (Effect)
+import Font.Style as FontStyle
+import Language.Strings (getString)
+import Language.Types (STR(..))
+import Styles.Colors as Color
+
+emptyTextView :: forall w. PrestoDOM (Effect Unit) w
```

Comment:
Can we create a file for genericHelperViews and move the common ones there as `emptyTextView` are being use in every view file